ui/tests/integration/components/replication-primary-card-test.js

```diff
@@ -1,13 +1,13 @@
 import { module, test } from 'qunit';
 import { setupRenderingTest } from 'ember-qunit';
 import { render } from '@ember/test-helpers';
-import engineResolverFor from 'ember-engines/test-support/engine-resolver-for';
+import { setupEngine } from 'ember-engines/test-support';
 import { CLUSTER_STATES } from 'core/helpers/cluster-states';
 import hbs from 'htmlbars-inline-precompile';
-const resolver = engineResolverFor('replication');
 
 module('Integration | Component | replication-primary-card', function (hooks) {
-  setupRenderingTest(hooks, { resolver });
+  setupRenderingTest(hooks);
+  setupEngine(hooks, 'replication');
```

Comment:
[New pattern for engine component tests](https://github.com/ember-engines/ember-engines/pull/653).